### DISCUSSION TITLE
Plot ODF

### DISCUSSION THREAD

**Discussion by ghost** [mtex-toolbox/mtex#701] [Ask Anything] [answered]
Hi all,

I plotted an ODF and realized that it is inverted in phi1 as compared with the ODF plotted in OIM. 

`odf = calcDensity(ebsd('Iron (Alpha)').orientations)

odf.SS = specimenSymmetry('222')
odf.CS = crystalSymmetry('432')

plot(odf,'phi2',45*degree, 'contour')

colorbar`

What did I do wrong?

**Comment by ralfHielscher** [comment, marked as answer]
Hi Arianne,

Have you read through this: https://mtex-toolbox.github.io/EBSDReferenceFrame.html

Ralf.

**Reply by ghost** [reply]
Thanks, Half!
I reviewed Reference Frame Alignment. Now it's OK!

**Reply by ralfHielscher** [reply]
Great  :smile:
